_posts/2023-07-15-gams-blog.md

```diff
@@ -0,0 +1,473 @@
+---
+layout: post
+title: "JuMP, GAMS, and the IJKLM model"
+date: 2023-07-15
+categories: [tutorials]
+---
+A [recent blog post](https://www.gams.com/blog/2023/07/performance-in-optimization-models-a-comparative-analysis-of-gams-pyomo-gurobipy-and-jump)
+by GAMS demonstrated a significant performance difference between JuMP and
+GAMS on a model they call IJKLM. The purpose of this blog post is to explain
+the difference and present a different formulation with much better
```

Comment:
maybe: “this performance difference and to present an alternative {compact, readable} modeling implementation in JuMP with much better performance”. 

Essentially I would suggest avoiding the word “formulation” as AIUI you are not creating different models for the same problem but instead only different ways of building the same model (modulo row/column permutations). I’d also make clear that we are only focusing on the JuMP implementation. Also it might be good to highlight that the best solution you present is quite readable and not overly complex.